scrabble.js

```diff
@@ -0,0 +1,34 @@
+class Scrabble {
+
+  constructor(word) {
+    this.word = word.toUpperCase();

```

Comment:
Make this `this.word = word?.toUpperCase();` so it only attempts to call `toUpperCase()` if `word` is indeed a **string**